### DISCUSSION TITLE
Why webpack-preprocessor use a pinned of @babel/parser ?

### DISCUSSION THREAD

**Discussion by alaingiller** [cypress-io/cypress#24917] [General]
Hi,

I saw the the webpack-preprocessor use a (very old) pinned version of @babel/parser, but not of @babel/core: 
https://github.com/cypress-io/cypress/blob/develop/npm/webpack-preprocessor/package.json#L24

Is there a reason why @babel/parser is pinned to this specific version "7.13.0" ? 
I think it should be the same as for the core "^7.0.1" or at least to allow patch version.

This break our pipeline because the version 7.13.0 is considered too old and there is a lot of patches and minor version since then.

**Comment by emilyrohrbough** [comment]
@alaingiller can you log this as an issue?

**Reply by alaingiller** [reply]
https://github.com/cypress-io/cypress/issues/24923
